crates/contracts/src/bin/vendor.rs

```diff
@@ -118,6 +118,10 @@ fn run() -> Result<()> {
             "balancer-labs/balancer-v2-monorepo/903d34e491a5e9c5d59dabf512c7addf1ccf9bbd/\
             pkg/deployments/tasks/20220609-stable-pool-v2/abi/StablePoolFactory.json",
         )?
+        .github(
+            "CoWSwapOnchainOrders",
+            "cowprotocol/ethflowcontract/v0.0.0/rc-artifacts/artifacts/CoWSwapOnchainOrders.sol/CoWSwapOnchainOrders.json"
```

Comment:
ubernit: Do you mind splitting up the line, Rust fmt struggles with these long lines:
```suggestion
            "cowprotocol/ethflowcontract/v0.0.0/\
             rc-artifacts/artifacts/CoWSwapOnchainOrders.sol/CoWSwapOnchainOrders.json"
```